src/System.Linq.Expressions/tests/Unary/UnaryConvertTests.cs

```diff
@@ -102,6 +102,18 @@ public static void ConvertArrayToIncompatibleTypeTest(bool useInterpreter)
             }
         }
 
+        [Fact]
+        public static void ToStringTest()
+        {
+            // NB: Unlike TypeAs, the output does not include the type we're converting to
+
+            var e1 = Expression.Convert(Expression.Parameter(typeof(object), "o"), typeof(int));
+            Assert.Equal("Convert(o)", e1.ToString());
```

Comment:
Looks like a bug. Perhaps should fix together wit **/pow/^ thing. 
I think string output has lower compat threshold since the most common use is dumping for debug purposes.